src/components/sharing/ShareBox/__tests__/ShareBox-test.js

```diff
@@ -1,23 +1,22 @@
-'use strict';
+jest.disableAutomock();
 
-jest.autoMockOff();
+import React from 'react';
+import ShareBox from '../';
+import ShareIcon from '../../ShareIcon/';
+import TestUtils from 'react-addons-test-utils';
 
 describe('Share Box', function() {
-  var React                  = require('react');
-  var ShareBox               = require('../');
-  var ShareIcon              = require('../../ShareIcon/');
-  var TestUtils              = require('react-addons-test-utils');
-  var scryRenderedComponents = TestUtils.scryRenderedComponentsWithType;
+  let scryRenderedComponents = TestUtils.scryRenderedComponentsWithType;
 
-  var mockServices = [
+  let mockServices = [
```

Comment:
If it's cool I'd rather leave this for future PR's (to get this over the line). That said I agree, I tend to prefer using `const` for the most part and only using `let` when necessary.